scripts/runtests.sh

```diff
@@ -169,7 +169,7 @@ python3 scripts/check-yaml-fields.py "$tmpdir"/var/log/installer/curtin-install/
 python3 scripts/check-yaml-fields.py <(python3 scripts/check-yaml-fields.py $tmpdir/etc/cloud/cloud.cfg.d/99-installer.cfg datasource.None.userdata_raw) \
         locale='"en_GB.UTF-8"' \
         timezone='"Pacific/Guam"' \
-        ubuntu_advantage.token='"C1NWcZTHLteJXGVMM6YhvHDpGrhyy7"' \
```

Comment:
Encoding a token publicly in a repo gave me pause, but Just confirmed that this token is, in fact, invalid/not real and cannot be used to attach.